### PR TITLE
add feature flag to perspective switcher

### PR DESCRIPTION
**Pull request by karthikjeeyar**
The perspective switcher will be enabled only if the dev console operator is installed.
When dev console is disabled, the perspective switcher should collapse/expand on clicking the hamburger menu.

### CODE REVIEW

**Comment by christianvogt** [comment]
Since the entire perspective switcher and mega menu are being worked on. I'm ok without tests for this change as the current perspective switcher is a temporary implementation.

**Review by christianvogt — APPROVED** [review]

**Comment by karthikjeeyar** [comment]
@christianvogt  @divyanshiGupta I have retained the existing functionality (collapse/expand of sidebar) when dev console is disabled.

**Comment by karthikjeeyar** [comment]
@rohitkrai03  I resolved the conflicts and rebased the PR. Please review the changes

File: frontend/public/extend/devconsole/shared/components/PerspectiveSwitcher.tsx [hunk 1]
```diff
@@ -52,28 +87,12 @@ export const PerspectiveSwitcher: React.FunctionComponent<Props> = (props: Props
...
-          >
-            <img
-              src={openshiftIconImg}
-              alt="Developer"
-              className="devops-perspective-switcher__openshift-logo"
-            />{' '}
-            Developer
-          </NavLink>
-        </li>
+        {devconsoleItem(props)}
```
Comment (rohitkrai03) [inline frontend/public/extend/devconsole/shared/components/PerspectiveSwitcher.tsx]: You are sending props as a parameter to `devConsoleItem()` but the function doesn't accept any parameters. Its using props from its parent scope.

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]

**Review by rohitkrai03 — APPROVED** [review]
LGTM

**Pull request merged**